web/src/components/trace/ObservationPreview.tsx

```diff
@@ -138,64 +144,49 @@ export const ObservationPreview = (props: {
               key={preloadedObservation.id}
             />
           ) : null}
+          {isScoreAttached && (
+            <Tabs value={selectedTab} onValueChange={setSelectedTab}>
+              <TabsList>
+                <TabsTrigger value="preview">Preview</TabsTrigger>
+                <TabsTrigger value="scores">Scores</TabsTrigger>
+              </TabsList>
+            </Tabs>
+          )}
         </div>
       </CardHeader>
       <CardContent className="flex flex-col gap-4">
-        <IOPreview
-          key={preloadedObservation.id + "-input"}
-          input={observationWithInputAndOutput.data?.input ?? undefined}
-          output={observationWithInputAndOutput.data?.output ?? undefined}
-          isLoading={observationWithInputAndOutput.isLoading}
-        />
-        {preloadedObservation.statusMessage ? (
-          <JSONView
-            key={preloadedObservation.id + "-status"}
-            title="Status Message"
-            json={preloadedObservation.statusMessage}
-          />
-        ) : null}
+        {selectedTab === "preview" && (
+          <>
+            <IOPreview
+              key={preloadedObservation.id + "-input"}
+              input={observationWithInputAndOutput.data?.input ?? undefined}
+              output={observationWithInputAndOutput.data?.output ?? undefined}
+              isLoading={observationWithInputAndOutput.isLoading}
+            />
+            {preloadedObservation.statusMessage ? (
+              <JSONView
+                key={preloadedObservation.id + "-status"}
+                title="Status Message"
+                json={preloadedObservation.statusMessage}
+              />
+            ) : null}
+            {observationWithInputAndOutput.data?.metadata ? (
+              <JSONView
+                key={observationWithInputAndOutput.data.id + "-metadata"}
+                title="Metadata"
+                json={observationWithInputAndOutput.data.metadata}
+              />
+            ) : null}
+          </>
+        )}
 
-        {observationWithInputAndOutput.data?.metadata ? (
-          <JSONView
-            key={observationWithInputAndOutput.data.id + "-metadata"}
-            title="Metadata"
-            json={observationWithInputAndOutput.data.metadata}
+        {selectedTab === "scores" && isScoreAttached ? (
+          <ScoresTablePreview
```

Comment:
is it possible to reuse the existing scores datatable? it already has pagination, filters, sorting etc implemented.

necessary props to pass in order to make it work
- column ids to show as we need to hide irrelevant ids such as traceId
- current traceid/observation id as a default filter. could be implemented in a similar way as on the user tabs which also reuse the main scores/trace/observation tables